.buildkite/scripts/steps/artifacts/cloud.sh

```diff
@@ -69,6 +69,9 @@ export CLOUD_DEPLOYMENT_ELASTICSEARCH_URL=$(ecctl deployment show "$CLOUD_DEPLOY
 echo "Kibana: $CLOUD_DEPLOYMENT_KIBANA_URL"
 echo "ES: $CLOUD_DEPLOYMENT_ELASTICSEARCH_URL"
 
+# Disable ansi color output for Node.js as we want to get plain values when executing node as a script runner below
+export FORCE_COLOR=0
```

Comment:
Nit: I think `NODE_DISABLE_COLORS=1` is technically more correct to use as I read [the docs](https://nodejs.org/docs/latest-v18.x/api/cli.html#node_disable_colors1), even `FORCE_COLOR=0` will have the same effect

Reply:
I've used `FORCE_COLOR` because I also found this on the docs: `When FORCE_COLOR is used and set to a supported value, both the NO_COLOR, and NODE_DISABLE_COLORS environment variables are ignored.`

We are setting `FORCE_COLOR` in previous ci setup scripts